src/hooks/sdk/useMediaStreamMetrics.tsx

```diff
@@ -64,21 +62,19 @@ export function useMediaStreamMetrics() {
           );
         }
 
+        // TODO: remove if condition after JS SDK 3.0.0-beta.2 is released
         if (clientMetricReport.getObservableVideoMetrics) {
```

Comment:
Why do we have to wait?

Reply:
So we remove the `ClientMetricReport` interface where the `getObservableVideoMetrics()` is optional. That change is not released yet and should be released in beta.2, I assume.